iocore/eventsystem/P_UnixSocketManager.h

```diff
@@ -55,21 +55,52 @@ transient_error()
   return transient;
 }
 
+#if !HAVE_ACCEPT4
+int
+accept4(int sockfd, struct sockaddr *addr, socklen_t *addrlen, int flags)
+{
+  int fd, err;
```

Comment:
How does this play with errno? I got specific feedback about saving and restoring errno around `close()`.